src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -139,4 +139,33 @@ public int getTotalMessageCount() {
     PreparedQuery results = datastore.prepare(new Query("Message"));
     return results.countEntities(FetchOptions.Builder.withLimit(1000));
   }
+
+  //Stores the user data
+  public void storeUser(User user) {
+	  Entity userEntity = new Entity("User",user.getEmail());
+
+	  userEntity.setProperty("email",user.getEmail());
+	  userEntity.setProperty("aboutMe",user.getAboutMe());
+	  datastore.put(userEntity);
+
```

Comment:
nit: remove the extra line breaks at the end of the function

Reply:
Fixed